docs/community/dev-card.md

```diff
@@ -0,0 +1,17 @@
+---
+id: dev-card
+title: 'Showcase your open source stats with your dev card'
+sidebar_label: 'Dev Card'
+keywords:
+- dev-card 
+---
+
+The Dev card feature is what we like to call your "open source business card." At the front, you see your username, the number of pull requests you have created, repositories you contribute to, a `View Profile` button that takes you to your dashboard, and a green icon, which describes your activity rate.
+![screenshot of front of a dev card](../../static/img/Group%20113.svg).
+
+When you click on the back of your card, you see additional information, including your OpenSauced profile's headline, amount of time you've been using our product, and your [PR velocity rate](../../docs/community/pr-velocity.md).
+! [screenshot of back of a dev card](../../static/img/devcard%20vbase%20back.svg).
```

Comment:
There's a space between the ! and [ that needs to be removed

Reply:
Done @BekahHW